File: hibernateTest/src/main/java/com/example/demo/Notabarista.java
package com.example.demo;


import lombok.Data;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import java.io.Serializable;

@Entity
@Table(name = "notabarista")
@Data
public class Notabarista {


    @Id
    @Column(name = "id", nullable = false)
    private Integer id;

    @Column(name = "name", nullable = false)
    private String name;

    public Notabarista(int id, String name) {
        this.id = id;
        this.name = name;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }


}

File: helloworld/src/main/kotlin/com/test/helloworld/Notabarista.kt
package com.test.helloworld

import lombok.Data
import java.io.Serializable
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "notabarista")
@Data
class Notabarista : Serializable {
    @Id
    @Column(name = "id", nullable = false)
    var id: Int? = null

    @Column(name = "name", nullable = false)
    var name: String? = null


    companion object {
        const val serialVersionUID = 1L
    }

    constructor(id: Int?, name:String?){
        this.id = id
        this.name = name
    }
}